user_guide_src/source/database/queries/020.php

```diff
@@ -1,3 +1,3 @@
 <?php
 
-$pQuery->close();
+$bool = $pQuery->close();
```

Comment:
The variable is unused.

Reply:
OK, I've completed the example query to better reflect the new mode of operation